src/pages/AdminPage/components.tsx

```diff
@@ -832,23 +890,33 @@ export const ContributorsTable = ({
                       key={`user-name-${u.id}`}
                       area="wide"
                       align="left"
+                      disabled={u.deleted_at !== null}
                     >
                       <UserName user={u} />
                     </Table.Cell>
 
-                    <Table.Cell key={`address-${u.id}`}>
+                    <Table.Cell
+                      key={`address-${u.id}`}
+                      disabled={u.deleted_at !== null}
```

Comment:
I'm bikeshedding here... but you could have avoided adding all these `disabled` lines and put it in the Row tag once instead.

Reply:
Pardon me. I tried to add it to the whole row. But we have a single column that is not disabled. And a child cannot have higher opacity than its parents, and workaround would make it complex.

Reply:
Ah, sorry, I overlooked the non-disabled cell. How about something like:

```
<Row css={isCoordinapeUser ? {
  '> td': { opacity: 0.3 },
  '> td.normal': { opacity: 1 }
}: null}>
  <Cell>a</Cell>
  <Cell className="normal">b</Cell>
</Row>
```